c/tskit/tables.h

```diff
@@ -684,6 +686,9 @@ typedef struct {
 /** @brief Do not run integrity checks before performing an operation. */
 #define TSK_NO_CHECK_INTEGRITY (1u << 29)
 
+/** @brief Do not reorder the population table. */
+#define TSK_UNCHANGED_POPULATIONS (1u << 28)
```

Comment:
I think this might want to be a common option - and myabe even default, since changing population references could be a big gotcha.